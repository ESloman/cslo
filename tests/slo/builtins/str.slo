print(str(nil));
print(str(false));
print(str(true));
print(str(0));
print(str(12.5));
