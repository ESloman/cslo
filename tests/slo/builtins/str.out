nil
false
true
0
12.5
